src/renderer/themes.css

```diff
@@ -237,6 +242,25 @@ it can be safely elided. This looks quite pleasant on this theme. */
   text-decoration: underline;
 }
 
+.nordic {
+  --primary-text-color: #EEEEEE;
+  --secondary-text-color: #ddd;
+  --tertiary-text-color: #EEEEEE;
+  --title-color: #EEEEEE;
+  --bg-color: #2b2f3a;
+  --favorite-icon-color: #85a4c1;
```

Comment:
**issue (blocking):** After rebasing with `development`, I can now see that the favorite icon color [is at an inaccessible color contrast](https://coolors.co/contrast-checker/eeeeee-85a4c1) with the `#EEEEEE` color it holds for an unfavorited video. This needs to be at least 3:1 to match WCAG standards.

![Screenshot_20240201_082351](https://github.com/FreeTubeApp/FreeTube/assets/84899178/bc3b9055-a55a-45fa-b9d4-7097d0cee801)

Reply:
On the color contrast test you linked, it is comparing the two favorite icon colors (#EEEEEE for unfavorited, #85a4c1 when it is favorited). Shouldn't it be #EEEEEE and #85a4c1 on a #2e3440 background?

Reply:
@JonahFoster You're technically correct for the WCAG 2.0 standard, but 3:1 as a minimum for differentiating the states of the same element [is forthcoming](https://accessibleweb.com/question-answer/what-are-the-contrast-requirements-for-an-elements-focus-mouseover-select-states/):
> When designing an interactive element, it needs to meet 3:1 contrast ratio with adjacent colors. You need to maintain 3:1 contrast ratio during all of that elements states (focus, hover, active, etc). The requirement doesn’t mean that states need to contrast 3:1 with each other. For instance, an element's hover state does not need to contrast 3:1 with its focus state, or even its resting state. Requirements like this are beginning to take shape with [WCAG 2.2, 2.4.11 Focus Visible (Enhanced)](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-enhanced). For now, you just need to make sure that each state of an element meets the 3:1 contrast ratio with adjacent colors, and are “distinguishable enough” from each other so users are aware of their interactions with the element (focusing, hovering, etc).

More subjectively speaking, favorited videos should ideally stick out to make it visually apparent from a distance which videos are saved, but making it very close to the main theme color mutes that effect. We do have other write-overs of the favorite color styling for a handful of the other themes @efb4f5ff-1298-471a-8973-3d47447115dc, but I don't think we should.

Reply:
Gotcha I see, that’s good to know thank you @jasonhenriquez. Would removing the favorite icon color line solve the problem? I can do that next time I’m at my computer.